test/demo/testUtils.test.ts

```diff
@@ -0,0 +1,35 @@
+import 'jest';
+import {OFPuppeteerBrowser, TestWindowContext} from './utils/ofPuppeteer';
+
+
+
+// Testing test utils...
+describe('basic puppeteer functionality', () => {
+  jest.setTimeout(100000);
```

Comment:
You'd think it would be easy to specify this in config, but apparently not. It was in every file because of an oversight when copy-pasting some boiler-plate.

I've changed it to only be in tests which we expect to take longer (which at the moment is just the context listener test since it creates apps).